env-defaults

```diff
@@ -60,6 +60,10 @@ CALLBACK_URL_ROOT=http://harmony:3001
 # The CMR Endpoint to use (e.g. URL for local, SIT, UAT, or production)
 CMR_ENDPOINT=https://cmr.uat.earthdata.nasa.gov
 
+# The Metrics Endpoint to use (e.g. URL for SIT, UAT, or production)
+# Should contain placeholders like {{query}}, {{from}}, and {{to}}, which are replaced at runtime.
+METRICS_ENDPOINT=http://localhost:3000?query={{query}}&from={{from}}&to={{to}}
```

Comment:
I think the endpoint should be just a URL - any type of construction should ideally happen in the code. It looks like you might need one more configuration item which is the name of the index to query?